edenai_apis/apis/perplexityai/perplexityai_api.py

```
@@ -121,3 +134,165 @@ def text__chat(
+            model_list=model_list,
+            drop_invalid_params=drop_invalid_params,
+            user=user,
+            modalities=modalities,
+            audio=audio,
+            **kwargs,
+        )
+        return response
+
+    async def llm__achat(
+        self,
+        messages: List = [],
```

Ruff (0.13.3)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)